azure-posix.yml

```diff
@@ -114,7 +113,6 @@ jobs:
       - bash: |
           set -e
           if [ $ANACONDA_ORG == "scipy-wheels-nightly" ]; then
-            source extra_functions.sh
             for f in wheelhouse/*.whl; do rename_wheel $f; done
```

Comment:
The if statement should probably be deleted.

Reply:
Ah yes. Me and bash aren't friends:(

Reply:
I note there is also a `rename_wheel` in `.travis.yml`.

Reply:
Correction, only `for f in wheelhouse/*.whl; do rename_wheel $f; done` still needs to be removed. The if also encompasses the upload.

Reply:
It didn't encompass the upload, all removed now.